templates/buildkite-elastic.yml

```diff
@@ -109,6 +111,16 @@ Parameters:
     Type: Number
     Default: 0
 
+  ScaleOutAdjustment:
+    Description: The number of agents to add on each scale out event
+    Type: Number
+    Default: 5
+
+  ScaleInAdjustment:
+    Description: The number of agents to remove on each scale in event
```

Comment:
Sorry, I was also meant to say, what do you think about describing the conditions here… just so people know what they're tweaking?